src/components/backend-ai-storage-list.ts

```diff
@@ -451,6 +452,11 @@ export default class BackendAiStorageList extends BackendAIPage {
           margin:15px 10px;
         }
 
+        div.align-top-center {
+          position: absolute;
```

Comment:
I'm sorry to reverse my initial offer, but it looks good to leave the button in the top right corner. Instead, how about aligning each button's icon and text to be vertical-align center and placing it in the center of the x button?

Reply:
I'll try to work on your guide after Lunchtime.

Reply:
Hmm, Unlike English, Indonesian(Bahasa Indonesia), Korean, For French, some menu button word length is too long, So It can only be centered at least 1400px or higher
* French 
<img width="1167" alt="image" src="https://user-images.githubusercontent.com/20002/165482142-8713b0d6-c0ab-4d3e-a707-730c5dfaf160.png">

* Russian 

<img width="1456" alt="image" src="https://user-images.githubusercontent.com/20002/165482732-5f4c85c3-7722-4eb3-90ac-291067d67964.png">

* Mongolian
<img width="1385" alt="image" src="https://user-images.githubusercontent.com/20002/165482944-2c3eb7e7-fb72-4380-9c52-38b646934bea.png">

As a result, I modified to center alignment only at 1400px and above.